mis_builder/models/mis_builder.py

```diff
@@ -190,17 +171,17 @@ def render_comparison(self, lang_id, value, base_value,
                 return self._render_num(
                     lang_id,
                     value - base_value,
-                    self.divider, self.dp, self.suffix, sign='+')
+                    self.divider, self.dp, '', self.suffix, sign='+')
```

Comment:
But in this specific comparison, it does make sense to keep the prefix. It's the case where we show the absolute difference: dollar minus dollar is still dollar.